src/dials/algorithms/indexing/bravais_settings.py

```diff
@@ -190,7 +196,9 @@ def lowest_symmetry_space_group_for_bravais_lattice(
     ).group()
 
 
-def refined_settings_from_refined_triclinic(experiments, reflections, params):
+def refined_settings_from_refined_triclinic(
+    experiments, reflections, params, cb_op_to_primitive=None
```

Comment:
Is `cb_op_to_primitive` not always set?

Even if it is `a,b,c` or `h,k,l` or whatever? I.e. a no-op?

Reply:
As above, this was mainly for backwards compatibility.